garrysmod/lua/includes/modules/weapons.lua

```diff
@@ -58,7 +58,7 @@ function Register( t, name )
 	list.Set( "Weapon", name, {
 		ClassName = name,
 		PrintName = t.PrintName or name,
-		Category = t.Category or "Other",
+		Category = t.Category or "#general_other_noname",

```

Comment:
`general_other_noname` looks silly. Probably best to just have it be `spawnmenu.category.other`.